packages/@aws-cdk/aws-codeguruprofiler/lib/profiling-group.ts

```diff
@@ -0,0 +1,86 @@
+import { Grant, IGrantable } from '@aws-cdk/aws-iam';
+import { Construct, Resource } from '@aws-cdk/core';
+import { CfnProfilingGroup } from './codeguruprofiler.generated';
+
+/**
+ * Properties for creating a new Profiling Group.
+ */
+export interface ProfilingGroupProps {
+
+  /**
+   * A name for the profiling group.
+   */
+  readonly profilingGroupName: string;
+
+}
+
+/**
+ * A new Profiling Group.
+ */
+export class ProfilingGroup extends Resource {
+
+  /**
+   * The name of this ProfilingGroup
+   */
+  public readonly profilingGroupName: string;
+
+  /**
+   * The ARN of this ProfilingGroup
+   */
+  public readonly profilingGroupArn: string;
+
```

Comment:
Missing `ProfilingGroup.fromProfilingGroupName` and `fromProfilingGroupArn`

Reply:
not very experienced with the implementation of such methods, i've given it my best guess.